src/OpenApi/Paths.hs

```diff
@@ -165,6 +208,10 @@ instance JSON.FromJSON Template where
 instance JSON.FromJSONKey Template where
    fromJSONKey = JSON.FromJSONKeyTextParser parseTemplateText
 
+instance JSON.ToJSON Template where
+  toJSON (Template segments)
+    = JSON.toJSON . ("/" <>) . Text.intercalate "/" $ toText <$> segments
```

Comment:
Could a form of fold be used here instead of `Text.intercalate`, allowing you to prepend `"/"` to the front of each element in `segments`?

Reply:
Possibly, you want to block on it? If this where more critical code I'd use a `Text.Builder` anyway.